### ISSUE TITLE
Unexpected token var with styled-components

### ISSUE THREAD

**Issue by donifer** [FormidableLabs/react-live#158]
I'm trying to use `react-live` with `noInline={false}` so that I can easily use react hooks, however when tied with a tagged template like `styled-components` I get a:

>SyntaxError: Unexpected token var

And I can't figure out why, what am I missing?

Here is a [CodeSanbox](https://codesandbox.io/s/styled-components-and-react-live-txnwy)

**Comment by sofiapoh** [comment]
Hey! Looks like you have a couple problems with the code I could identify from a quick look: 
- `Text` is defined inside your React component when it should be defined outside the component using your `styled-component`
- `testComponent` should be PascalCase
- in `Text` you're not setting any specific value to be red or blue, you could fix this by adding `color: ${props => isRed ? 'red' : 'blue};`
- Currently we expect only only single component or expression when using the `noInline={false}` prop (this is the default), in this case you have two. If you'd like to render the `TestComponent` which uses `Text` you'd have to use the `noInline` prop. 

I realise the last case is bad documentation in my part! I'll make a PR to improve documentation around the `noInline` default behaviour! Hope this helps.

**Comment by donifer** [comment]
Thanks for your reply @sofiapoh!

Sorry with my rushed example, I updated it with your suggestions.

I'm defining `Text` inside since I want that to actually show (image documenting how to use styled components or something similar).

I added a second example of it working if I don't use a tagged template literal. So I think its something going on of how it's being handled by buble since I get a different output from it when using render vs not using something like:
`!functionvar templateObject = ...`
vs
`var templateObject = ...`

Tagged template literals work if I use `render` however if I use `render` I don't think I can use react hooks.
